### DISCUSSION TITLE
Discussions Topics for Nov 15, 2021 Engineering Sync

### DISCUSSION THREAD

**Discussion by clediggins-usds** [CDCgov/prime-reportstream#3006] [Show and tell]
Discussion Topics for Nov 15, 2021 Engineering Sync

**Comment by clediggins-usds** [comment]
From SoS 11/10 : 
Discussions around standardized PR review and approval process for all of ReportStream to reduce inter-team blockage.  As the team(s) grow this could become a larger problem.

* How many approvals are needed?
* Do we have code owners / final stamp of approval?
* How to handle when 'approvers' are OOO?
* PR turnaround time expectations

**Comment by carlosfelix2** [comment]
Code coverage with code viewer

**Comment by RickHawesUSDS** [comment]
I'd like to talk about M1 Macs.
